packages/create-bison-app/template/config.ts

```diff
@@ -0,0 +1,97 @@
+import { z } from 'zod';
+
+import { notEmpty } from './lib/type-witchcraft';
+
+const stages = ['production', 'staging', 'development', 'test'] as const;
```

Comment:
This is more of a question about when `staging` should be used and the purpose of when it should be used in the codebase. In your experience, do you set `NODE_ENV=staging` for any deploys that are not production? I've always used `production` for these.

Reply:
Good point. We probably won't use staging on most apps.